postman/environments/local.postman_environment.json

```diff
@@ -39,7 +39,19 @@
 			"enabled": true
 		},
 		{
-			"key": "HearingId",
+			"key": "HearingResultsId",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{	
+			"key": "hearing_id",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "hearing_date",
```

Comment:
> Please can we include working examples of these variables in the default values?

I'll have a try. I'll check to see if the hearing id is seeded and if there's one that isn't set by the DB as this means it could fail.

Reply:
The hearing_id provided in local depends on what's created by the HMCTS service. Locally, it'll need to be passed in by the developer i think